inception-imls-opennlp/src/test/java/de/tudarmstadt/ukp/inception/recommendation/imls/opennlp/doccat/OpenNlpDoccatRecommenderTest.java

```diff
@@ -82,8 +83,7 @@ public void setUp()
     }
 
     @Test
-    public void thatTrainingWorks() throws Exception
-    {
+    public void thatTrainingWorks() throws IOException, UIMAException, RecommendationException {
```

Comment:
Please avoid changes not relevant to the issue. For new code or modified code, please use the DKPro Core code style. An Eclipse formatter configuration file is attached here: https://dkpro.github.io/contributing/